balancer-labs/frontend-v2
# New featured protocols section on homepage

# Description

Updated the featured section from featured Element pools to 'Featured Protocols'.
I've kept the 'FeaturedPools' component and created a new one for FeaturedProtocols, in case we want to bring back the Featured Pools section at a later stage.

## Type of change


## How should this be tested?
1. Go to the home page and view the 'Featured Protocols' section.

It should look like this on desktop:

<img width="1322" alt="Screen Shot 2022-05-05 at 1 01 50 PM" src="https://user-images.githubusercontent.com/1121139/166919105-afbd8fc1-0afe-40e9-9937-b7f1c56cc357.png">

## Checklist:

- [] I have requested at least 2 reviews (If the PR is significant enough, use best judgement here)
